Src/Sarif.PatternMatcher/AnalyzeOptions.cs

```diff
@@ -26,23 +26,11 @@ public class AnalyzeOptions : AnalyzeOptionsBase
                        "performance and/or result in calls across the network).")]
         public bool DynamicValidation { get; internal set; }
 
-        [Option(
-            'm',
-            "multiline",
-            HelpText = "Enable multiline regular expression search.")]
-        public bool Multiline { get; internal set; }
-
         [Option(
             "deny-regex",
             HelpText = "A regular expression used to suppress scanning for any file or directory path that matches the regex.")]
         public string FileNameDenyRegex { get; internal set; }
 
-        [Option(
-            "file-size-in-kb",
-            HelpText = "The maximum file size (in kilobytes) that will be analyzed.",
-            Default = 1024)]
-        public int FileSizeInKilobytes { get; internal set; }
```

Comment:
>FileSizeInKilobytes [](http://example.com/codeflow?start=19&length=19)

this arg obsoleted by max-file-size-in-kb, which we now pick up from the driver framework.